packages/invoices/src/InboundShipment/DetailView/reducer/reducer.ts

```diff
@@ -197,9 +203,18 @@ export const reducer = (
             return itemsArray;
           }, draft.items);
 
+          break;
+        }
+        case ActionType.UpdateInvoice: {
+          const { payload } = action;
+          const { key, value } = payload;
+
+          state.draft[key] = value;
+
```

Comment:
just needed to add this to the draft. Wish there was a way to ensure the fn was non-optional